### ISSUE TITLE
Disabling extension doesn't turn off widget

### ISSUE THREAD

**Issue by crutkas** [microsoft/devhome#1390]
### Dev Home version

0.401.219.0

### Windows build number

10.0.25926.1001

### Other software

OS Build Version: 10.0.25926.1001.amd64fre.rs_we_adept.230805-1700
.NET Version: .NET 6.0.20

### Steps to reproduce the bug

1. disable extensions
2. Go to dashboard
3. go to "add widget"
4. See disabled extensions

![image](https://github.com/microsoft/devhome/assets/1462282/a5822048-1e94-4246-ba09-c15c31271659)
![image](https://github.com/microsoft/devhome/assets/1462282/8b655f53-8584-4e9a-a3a3-7d6ce9263cbf)

### Expected result

Extensions do not show up

### Actual result

Preview extensions show up while off

### Included System Information

Physical Memory: 31.84GB (17.34GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHome_0.301.198.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension_0.300.198.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHome.Canary_0.401.219.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension.Canary_0.400.219.0_x64__8wekyb3d8bbwe

**Comment by embender** [comment]
I am also seeing extensions appear within add widgets that are not within the extensions settings page, should I make this a separate issue? It feels related - 

![image](https://github.com/microsoft/devhome/assets/44589070/e8c672d4-dccc-4c6a-b87b-812e199478b2)
![image](https://github.com/microsoft/devhome/assets/44589070/05eccf2b-c3a2-461f-8999-0b5cd2cb8bcf)

**Comment by crutkas** [comment]
dev home widgets for that version (canary vs preview), won’t appear intentionally. Canary won’t shoe canary’s core widgets. Is that what you are referring to?

**Comment by embender** [comment]
not sure i understand what you're saying, I am saying that I see two options for extensions to turn off and then three different extensions within the "add widgets" area, with the discrepancy being with the Core (Preview) being an extension that appears within the add widgets, but not within the extension settings

**Comment by crutkas** [comment]
Good catch but No one except us would see this difference between canary and preview naming quirk here.   Also it would be a different bug.
